tests/h/views/api_groups_test.py

```diff
@@ -8,28 +8,36 @@
 from pyramid.httpexceptions import HTTPNoContent, HTTPBadRequest
 
 from h.views import api_groups as views
+from h.services.profile_group import ProfileGroupService
+from h.services.group import GroupService
 
 
 @pytest.mark.usefixtures('profile_group_service')
 class TestGroups(object):
     def test_groups_proxies_to_service(self, pyramid_request, profile_group_service):
         views.groups(pyramid_request)
 
-        assert profile_group_service.called_once()
+        profile_group_service.all.assert_called_once()
 
     def test_groups_passes_authority_parameter(self, pyramid_request, profile_group_service):
         pyramid_request.params = {'authority': 'foo.com'}
 
         views.groups(pyramid_request)
 
-        assert profile_group_service.called_once_with(pyramid_request, 'foo.com')
+        profile_group_service.all.assert_called_once_with(user=mock.ANY,
+                                                          authority='foo.com',
+                                                          document_uri=mock.ANY)
+
+        c_args, c_kwargs = profile_group_service.all.call_args
```

Comment:
This can be done in one line by accessing `call_args[1]['authority']` but I felt that the tuple unpacking would make this easier to follow for future test maintainers.